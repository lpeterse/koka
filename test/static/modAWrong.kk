module static/modAWrong {
  public type int
  public alias aint = int
}
