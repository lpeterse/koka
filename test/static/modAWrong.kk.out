static/modAWrong/aint: V
static/modAWrong/int: V
public alias aint = int = 1